lib/engine/game/g_1861.rb

```diff
@@ -14,15 +14,64 @@ class G1861 < G1867
                                 'After the minors and majors operates the national runs trains, '\
                                 'withholds and buys as many trains as possible'],
       ).freeze
+      GAME_LOCATION = 'Russia'
       GREEN_CORPORATIONS = %w[MB Y V TR SV E].freeze
 
+      # This is Kh in 1861
+      HEX_WITH_O_LABEL = %w[G15].freeze
+      HEX_UPGRADES_FOR_O = %w[201 202 207 208 622 623 801 640].freeze
+      BONUS_CAPITALS = %w[H8].freeze
+      BONUS_REVENUE = 'Q3'
+      NATIONAL_RESERVATIONS = %w[E1 H8].freeze
+
       def self.title
         '1861'
       end
 
       def unstarted_corporation_summary
-        summary, _corps = super
-        [summary, [@national]]
+        unipoed = @corporations.reject(&:ipoed)
+        minor = unipoed.select { |c| c.type == :minor }
```

Comment:
partition ?

Reply:
They're not quite the same since there's minor, major and national in the list, so partition wouldn't give the same outcome

I've modified it slightly since having the national first is probably worthwhile since it'll be run.